File: java/oop/CityCodeApp/src/com/techlabs/citycode/CityCode.java
package com.techlabs.citycode;

public class CityCode {
	private String cityCode;
	private String description;
	
	public CityCode(String cityCode,String description){
		this.cityCode=cityCode;
		this.description=description;
	}

	public String getCityCode(){
		return this.cityCode;
	}
	
	public String getDescription(){
		return this.description;
	}
}

File: dotNet/csharp/CityCode/CityCode/CityCode.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CityCode
{
    class CityCode
    {
        private string _cityCode;
        private string _description;

        public CityCode(string citycode,string desciption)
        {
            this._cityCode = citycode;
            this._description = desciption;
        }

        public string Code
        {
            get
            {
                return _cityCode;
            }
        }
        public string Desciption
        {
            get
            {
                return _description;
            }
        }
    }
}
